pkg/packagekit/updates.jsx

```diff
@@ -509,6 +515,116 @@ const AskRestart = ({ onIgnore, onRestart, history }) => <>
+            </FlexItem>
+        </Flex>
+    </Flex>);
+};
+
+class CardsPage extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            autoUpdatesEnabled: undefined,
+            autoUpdatesType: undefined,
+            autoUpdatesDay: undefined,
+            autoUpdatesTime: undefined,
+        };
+    }
+
+    render() {
+        const cardContents = [
+            {
+                id: "status",
+                title: _("Status"),
+                className: "pk-card-info",
+                actions: (<Tooltip content={_("Check for updates")}>
+                    <Button variant="secondary" onClick={this.props.handleRefresh}><RedoIcon /></Button>
+                </Tooltip>),
+                body: <StatusCard updates={this.props.updates}
+                                  highestSeverity={this.props.highestSeverity}
+                                  timeSinceRefresh={this.props.timeSinceRefresh} />
+            },
+            {
+                id: "automatic-updates",
+                className: "pk-card-info",
+                title: _("Automatic updates"),
+                actions: (<AutoUpdates onInitialized={newState => this.setState(newState)} privileged={this.props.privileged} />),
+                body: (<AutoUpdatesBody enabled={this.state.autoUpdatesEnabled}
+                                        type={this.state.autoUpdatesType}
+                                        day={this.state.autoUpdatesDay}
+                                        time={this.state.autoUpdatesTime} />),
+            },
+        ];
+
+        if (this.props.state === "available") { // automatic updates are not tracked by PackageKit, hide history when they are enabled
+            cardContents.push({
+                id: "available-updates",
+                title: _("Available updates"),
+                actions: (<div className="pk-updates--header--actions">
+                    {this.props.applySecurity}
+                    {this.props.applyAll}
+                </div>),
+                body: <UpdatesList updates={this.props.updates} />
+            });
+        }
+
+        if (!this.state.autoUpdatesEnabled && this.props.history.length > 0) { // automatic updates are not tracked by PackageKit, hide history when they are enabled
+            cardContents.push({
+                id: "update-history",
+                title: _("Update history"),
+                body: <History packagekit={this.props.history} />
+            });
+        }
+
+        return cardContents.map(card => {
+            return (
+                <Card key={card.id} className={card.className} id={card.id}>
+                    <CardHeader>
+                        <CardTitle><h2>{card.title}</h2></CardTitle>
+                        {card.actions && <CardActions>{card.actions}</CardActions>}
+                    </CardHeader>
+                    <CardBody>
```

Comment:
This needs the class 'contains-list'
In all other pages when we use tables inside card bodes we remove the margins.
I don't have an opinion if it's better or worse design, but it's really important that we are consistent in the usage of the cards.
So add here className='contains-list' and the result should be like the following screenshot (after your also remove the CSS rule with the added margin)

![Screen Shot 2020-12-07 at 10 14 23](https://user-images.githubusercontent.com/14921356/101332180-3ebf8500-3875-11eb-9c2e-ca9d0d0f28f9.png)